config/environments/test.rb

```diff
@@ -18,7 +18,7 @@
   # Configure public file server for tests with Cache-Control for performance.
   config.public_file_server.enabled = true
   config.public_file_server.headers = {
-    'Cache-Control' => "public, max-age=#{1.hour.seconds.to_i}"
+    'Cache-Control' => "public, max-age=#{1.hour.to_i}"
```

Comment:
Can I safely assume that the reason behind small random changes like this is "Rails 5.2 is not Rails 5.1"?

Reply:
Yup.